cmd/broker/modules.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/Azure/open-service-broker-azure/pkg/services/eventhubs"
 	"github.com/Azure/open-service-broker-azure/pkg/services/keyvault"
 	"github.com/Azure/open-service-broker-azure/pkg/services/mysqldb"
-	"github.com/Azure/open-service-broker-azure/pkg/services/postgresqldb"
+	"github.com/Azure/open-service-broker-azure/pkg/services/postgresql"
```

Comment:
This is the first place you'll spot a pervasive change. I changed the package name for the postgres module. Since the module has managers that handle both dbms and databases, it seemed that the name `postgresqldb` hinted too strongly as just handling databases. I wanted to fix that.